app/docker/components/log-viewer/logViewerController.js

```diff
@@ -36,4 +37,8 @@ function (clipboard) {
       this.state.selectedLines.splice(idx, 1);
     }
   };
+
+  this.getLogViewerCSSClass = function() {
```

Comment:
You can remove this function, see my comment above.

Reply:
fixed in 78dde81b